File: src/Java/BehavioralPatterns/Command/FastFoodRestaurant/src/Commands/ModifyMenuItemCommand.java
package Commands;

import Models.MenuItem;

import java.util.List;

public class ModifyMenuItemCommand extends OrderCommand {
    @Override
    public void execute(List<MenuItem> fastFoodOrderMenuItems, MenuItem menuItem) {
        for (MenuItem item : fastFoodOrderMenuItems) {
            if (item.getName().equals(menuItem.getName())) {
                item.setPrice(menuItem.getPrice());
                item.setAmount(menuItem.getAmount());
                break;
            }
        }
    }
}

File: src/PHP/BehavioralPatterns/Command/FastFoodRestaurant/Commands/ModifyMenuItemCommand.php
<?php

namespace BehavioralPatterns\Command\FastFoodRestaurant\Commands;

use BehavioralPatterns\Command\FastFoodRestaurant\Models\MenuItem;

class ModifyMenuItemCommand extends OrderCommand {
    public function execute(array &$fastFoodOrderMenuItems, MenuItem $menuItem): void
    {
        foreach ($fastFoodOrderMenuItems as &$fastFoodOrderMenuItem) {
            if ($fastFoodOrderMenuItem->name === $menuItem->name) {
                $fastFoodOrderMenuItem->price = $menuItem->price;
                $fastFoodOrderMenuItem->amount = $menuItem->amount;
                break;
            }
        }
    }
}
